source/scene/material.ts

```diff
@@ -0,0 +1,21 @@
+
+import { Bindable } from '../bindable';
+import { Program } from '../program';
+
+export abstract class Material implements Bindable {
```

Comment:
`Material` is not an OpenGL concept but I think `Bindable` was introduced for OpenGL concepts, mainly. What is the take on this?

Reply:
passt